packages/common/form-payload.ts

```diff
@@ -0,0 +1,88 @@
+export interface FormPayload {
+  personalInformation: {
+    firstName: string;
+    lastName: string;
+    postalCode: string;
+    primaryPhone: string;
+    secondaryPhone: string;
+    email: string;
+  };
+
+  skillInformation: {
```

Comment:
I would have expected the value of this to be equal to the class `SkillInformationDTO` defined in API, can we get away with reusing it? Is there a reason to duplicate the definition?

Reply:
I think it's a question of pattern and use. A DTO in the Nest ecosystem is mean to represent a received object, not the object itself. While we could technically reuse it, it would break nests patterns.

Reply:
Fair enough, I think I'm a bit confused on what exactly the shared parts are meant to be 🤔